azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -148,7 +148,7 @@ private void logRequest(final HttpServletRequest req, final Session session) {
     buf.append("\"");
     buf.append(req.getMethod()).append(" ");
     buf.append(req.getRequestURI()).append(" ");
-    if (req.getQueryString() != null) {
+    if (req.getQueryString() != null && allowedPostRequest(req)) {
```

Comment:
req.getQueryString() != null is not necessary since same check is already done in allowedPostRequest